web_api/web_api/migrations/0026_auto_20220322_0036.py

```diff
@@ -0,0 +1,53 @@
+# Generated by Django 3.0.3 on 2022-03-22 00:36
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('web_api', '0025_auto_20200902_0052'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='account',
+            name='created_at',
+            field=models.DateTimeField(auto_now_add=True, db_index=True),
```

Comment:
do these need to use concurrent stuff or is it nbd?

Reply:
It's probably okay if we don't